src/atoms/Switch/Switch.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react';
+
+import styled from '_styled-components';
+import { Input } from 'atoms';
+import Typography from 'Typography';
+/* stylelint-disable max-nesting-depth */
+/* stylelint-disable unit-whitelist */
+/* stylelint-disable no-descending-specificity */
+const LabelText = styled(Typography)`
+  font-size: 0.875em;
+  color: #163150;
+`;
+export const Checkbox = styled(Input)`
+  :checked + span {
+    background-color: #b2d7e0;
+  }
+  :checked + span::before {
+    transform: translateX(30px);
+  }
+`;
+const SwitchContainer = styled.div`
+  display: flex;
+  align-items: center;
+`;
+const SliderBackground = styled.label`
+  position: relative;
+  display: inline-block;
+  width: 44px;
+  height: 16px;
+  margin-left: 8px;
+  margin-right: 8px;
+  input {
+    display: none;
+  }
+`;
+const Slider = styled.span`
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: #b2d7e0;
+  transition: 0.4s;
+  border-radius: 17px;
+  ::before {
+    position: absolute;
+    content: '';
+    height: 22px;
+    width: 22px;
+    left: -4px;
+    bottom: -3px;
+    background-color: ${props => props.theme.primary};
+    transition: 0.4s;
+    border-radius: 17px;
+  }
+`;
+interface Props {
+  labelLeft?: string;
+  labelRight?: string;
+  handleChange?(): void;
```

Comment:
We also need a prop to control the value, similar to `<input type="checkbox" checked />`, so that the user can control the initial value and change it from outside the component. This should only require passing a `checked` prop to `Checkbox`, we still won't need state.

Reply:
Good plan! I added this to the `defaultChecked` prop